tests/support/conversions_test.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import torch
+from iree.turbine.support.conversions import torch_dtyped_shape_to_iree_format
+
+
+def test_torch_dtyped_shape_to_iree_format():
+    iree_format = torch_dtyped_shape_to_iree_format([1, 2, 3], dtype=torch.bfloat16)
+    assert iree_format == "1x2x3xbf16"
```

Comment:
Thanks for the unit test! Can you please also add cases for other formats? The list of test cases doesn't need to be exhaustive, but it should cover a few other representative cases and ideally error conditions

Should have a few other shapes and dtypes. Could parameterize the test case too https://docs.pytest.org/en/stable/how-to/parametrize.html

something like
```python
@pytest.mark.parametrize(
    "shape,dtype,expected",
    [
      ([1,2,3], torch.bfloat16, "1x2x3xbf16"),
      ([1,2,3], torch.float32, "1x2x3xf32"),
    ]
)
```

Reply:
I refactored the tests as you suggested.